msprime/trees.py

```diff
@@ -114,7 +116,9 @@ def __eq__(self, other):
 
 class Node(SimpleContainer):
     """
-    A :ref:`node <sec_node_table_definition>` in a tree sequence.
+    A :ref:`node <sec_node_table_definition>` in a tree sequence, corresponding
+    to a single genome. The ``time`` and ``population`` are attributes of the
+    ``Node``, rather than the ``Individual``, for historical reasons.
```

Comment:
this would be confusing, otherwise. maybe a longer explanation is in order?

Reply:
It's not entirely historical, as you explained elsewhere. Maybe just refer back to that for the explanation?